### ISSUE TITLE
 sickrage shows show as unaired, even past the air date, causing them not to download. -SHOWUPDATER not working

### ISSUE THREAD

**Issue by randmvilnce** [SiCKRAGETV/sickrage-issues#1043]
I originally posted this to SR forums, link below.

https://sickrage.tv/forums/forum/help-support/general-support/20748-sickrage-shows-show-as-unaired-even-past-the-air-date-causing-them-not-to-download

Basically, showupdater is not running,ever. i am having to manually go and search for each show in my daily download list.
no messages in logs about showupdater, even restarting SR does not initiate showupdater when i do have it set in settings.
manually clicking the search button, or doing a force full update from the shows page is the only way to get the shows to download.  even then sometimes force full update does not start the DL and i have to go back and click search. which finds it instantly.
Any and all help is appreciated.

my log is 15mb and too big to UL here, so i put it on a dropbox share.
https://www.dropbox.com/s/8t6iuey8px5tl88/sickrage.log?dl=0

running master release on Win7 64x
Anything else i can provide or do just let me know.
Thanks.

**Comment by fernandog** [comment]
@randmvilnce 

SHOWUPDATER is not running - maybe - because your log is FULL of POSTPROCESSER threads!
what is your post-processor frequency?

SickRage restarted or crashed in all this timestamps. Can you check windows log or something?

```
	Line 2515: 2015-03-25 13:40:20 INFO     MAIN :: Trying to clean cache folder C:\SickRage\cache
	Line 6769: 2015-03-25 13:47:58 INFO     MAIN :: Trying to clean cache folder C:\SickRage\cache
	Line 21818: 2015-03-25 19:38:03 INFO     MAIN :: Trying to clean cache folder C:\SickRage\cache
	Line 40287: 2015-03-26 03:19:23 INFO     MAIN :: Trying to clean cache folder C:\SickRage\cache
```

**Comment by randmvilnce** [comment]
the log has been set to 75MB 2 days ago, so its not a full log scenario.

post processing is set to 5, Shall i increase that?

the 3:19 entry on not sure, as I was asleep then. But SR was running when i got up this morning. And the other times mentioned would have been me restarting SR during testing.

**Comment by fernandog** [comment]
@randmvilnce 5 ?! that's crazy! Depending how many files you have to PP, it can take more than 5min! A new thread of PP will start before the old finishes

Set to at least 15min

**Comment by fernandog** [comment]
@TagForce another one...any news on the self.amActive for PP?

**Comment by TagForce** [comment]
@fernandog Still working on it :)

**Comment by fernandog** [comment]
@TagForce do you think that the flood of PP threads can block SHOWUPDATER thread?

**Comment by TagForce** [comment]
It shouldn't...
BTW... PP already waits for the last run to finish before starting a new one...
I've added a time.sleep(75) to the processDir function, so it'll wait 75 seconds before doing anything.
Set my PP frequency to 1 minute.
PP runs every 75 seconds.
Remove the time.sleep(75) and it runs every 60 seconds.
So the only thing that can happen simultaneously is a manual PP.

**Comment by TagForce** [comment]
time.sleep(75):
```
15:39:47 DEBUG::POSTPROCESSER :: Starting new thread: POSTPROCESSER
15:41:03 DEBUG::POSTPROCESSER :: Starting new thread: POSTPROCESSER
15:42:19 DEBUG::POSTPROCESSER :: Starting new thread: POSTPROCESSER
```
no sleep:
```
15:49:05 DEBUG::POSTPROCESSER :: Starting new thread: POSTPROCESSER
15:50:06 DEBUG::POSTPROCESSER :: Starting new thread: POSTPROCESSER
15:51:07 DEBUG::POSTPROCESSER :: Starting new thread: POSTPROCESSER
```

**Comment by randmvilnce** [comment]
I have it set to 15 now.
I'll check the logs again shortly.

**Comment by fernandog** [comment]
@TagForce so we don't need to check if PP is running before start another?

**Comment by randmvilnce** [comment]
![sr3](https://cloud.githubusercontent.com/assets/11664610/6849285/542d64be-d3b5-11e4-8fe8-014d5be9a31f.JPG)

when i search for showupdater this is the only reference I get

**Comment by TagForce** [comment]
@fernandog Nope.

**Comment by randmvilnce** [comment]
so what shall I try next gents?

**Comment by fernandog** [comment]
@randmvilnce you have to wait until 03:00am or change in the settings the time of showupdated to next hour

**Comment by randmvilnce** [comment]
ok I just set it to 2pm (14)  (its 1:55 here now) so ill report back in a few min, and attach a log again.

**Comment by fernandog** [comment]
@randmvilnce i think you must restart after change this

**Comment by randmvilnce** [comment]
ok i have changed it to 15, then restarted SR, so in 50 min or so ill report back and post a log.

**Comment by randmvilnce** [comment]
the time passed, but no showupdater occurred Log file attached.
I have debug mode set to 1 in SR config.ini as well 
ill ask again, would a backup/reinstall of SR and a restore maybe take care of this issue?

https://www.dropbox.com/s/u8j1atcv01ggupb/sickrage%20%282%29.log?dl=0
.
thanks.

**Comment by fernandog** [comment]
@randmvilnce I really don't know what is happening. don't know if a restore can fix

@abeloin any ideias? there's nothing in the log showing about SHOWUPDATER

**Comment by randmvilnce** [comment]
this circumstance happened right after i did the Sunday update, worked fine up until that point.

**Comment by fernandog** [comment]
@randmvilnce the strange thing is that only you are reporting this. nobody else.

**Comment by abeloin** [comment]
@randmvilnce Stop SR and start SR then look in the log for `Starting new thread`
Also what is the value of your showupdate_hour in your config.ini

**Comment by randmvilnce** [comment]
@abeloin 

showupdate_hour = 15 

attached is the log that shows "Starting new thread" 

![sr4](https://cloud.githubusercontent.com/assets/11664610/6854208/d06457dc-d3d2-11e4-833c-b77e9a208d71.JPG)

nothing in there about updater though.. im stumped..

**Comment by abeloin** [comment]
@randmvilnce Yes, you can try to backup and restore your db on a fresh copy of sickrage.

If it doesn't work(do this like before 15 min before the next hour):
Stop SR, set showupdate_hour to the next hour(in my case it's 14:45 so it would be 15) in config.ini. Start sickrage. Wait until it's past 20 mins to the hour, stop sickrage. Dump all sickrage log file to dropbox so we have the log from start, when it should start and after.

**Comment by randmvilnce** [comment]
here's what i did, i had an Acronis image from Sunday at 4Am, I backed up my SR DB, renamed my SR folder and replaced it with the one from the 22nd before the update. started SR back up imported my DB and everything is fine checked the logs and the updater is running. now my version is pre update, so ill wait until after tonight and see if it downloads work. if they do then ill wait till Sunday then do an update.
thanks for all the help.

**Comment by randmvilnce** [comment]
everything went fine last night, all shows downloaded.
I will report back on sunday after i update and we will see if the problem comes back or not.
thanks.

**Comment by fernandog** [comment]
@cyberundertaker continue post here please

**Comment by cyberundertaker** [comment]
@fernandog it looks like it's working, but actually it's not...

This is the log:
2015-03-27 03:29:26 INFO SHOWUPDATER :: Completed full update on all shows
2015-03-27 03:29:25 INFO SHOWUPDATER :: Trying to clean cache folder C:\SickRage-develop\cache
2015-03-27 03:29:25 INFO SHOWUPDATER :: Doing full update on all shows

And this is 10 hours after the timestamps in the log...This show is queued and awaiting an update.
![simp2](https://cloud.githubusercontent.com/assets/11670746/6867968/7a5cad70-d487-11e4-895c-b2bf9f228613.PNG)

all continuing shows show the same message

**Comment by abeloin** [comment]
@cyberundertaker Can you provider the debug log(sickrage.log)?

**Comment by cyberundertaker** [comment]
@abeloin This is the SHOWQUEUE-REFRESH log:
http://pastebin.com/P7hmeLkD

Full log(sickrage.log, 15MB):
https://drive.google.com/file/d/0ByFb16E-5XuQMGtmNDMzRGstMmM/view?usp=sharing

**Comment by cyberundertaker** [comment]
@abeloin @fernandog 
Any news why it's not updating shows?

**Comment by abeloin** [comment]
@cyberundertaker Can you disable the metadata generator and retry?

**Comment by cyberundertaker** [comment]
@abeloin ok, i disabled metadata creation during post processing completely and set update at 22pm (in 30 min). now we wait. will report back after

**Comment by cyberundertaker** [comment]
@abeloin same thing happens. it seems to be stuck at falling skies
![fs](https://cloud.githubusercontent.com/assets/11670746/6878225/875547be-d4da-11e4-9ff2-81cd207e7029.PNG)

sickrage.log(7MB)
https://drive.google.com/file/d/0ByFb16E-5XuQVl96UlRRTUJ0MmM/view?usp=sharing

**Comment by abeloin** [comment]
@cyberundertaker Can you temporary move this file Falling.Skies.S03E01E02.PROPER.HDTV.x264-EVOLVE outside of sickrage library? Then try again

**Comment by abeloin** [comment]
@cyberundertaker Can you tell me which subtitle providers are enable in your SR?

**Comment by cyberundertaker** [comment]
@abeloin 
tvsubtitles
thesubdb
opensubtitles

i removed Falling.Skies.S03E01E02.PROPER.HDTV.x264-EVOLVE and turned off subtitle search
i set the update at 2am (in 40 min), so will report back

**Comment by fernandog** [comment]
@cyberundertaker are you restarting after changing update hour?

**Comment by cyberundertaker** [comment]
@fernandog yes

**Comment by fernandog** [comment]
@cyberundertaker I just pushed a fix so you don't have to restart.

**Comment by fernandog** [comment]
@cyberundertaker any lucky? did it worked removing that file?

**Comment by DailyPlanet4** [comment]
So I am having the same issue. No shows will auto update and if they do they say UN-aired so they do not snatch. Any ideas? and please talk in layman's terms, I am not super technical :)

**Comment by cyberundertaker** [comment]
@fernandog @abeloin no luck again. still stuck at falling skies

**Comment by cyberundertaker** [comment]
@fernandog @abeloin i removed falling skies completely from SR and trying update in 40 min...
also will try cleaning up falling skies media files so there are only .mp4 in folders, and try to add it back to SR after update in 40min

**Comment by cyberundertaker** [comment]
after removing falling skies(didn't put them back) from SR and after the update it's stuck at Orphan Black...The episodes below are currently being refreshed from disk

full log:
https://drive.google.com/file/d/0ByFb16E-5XuQZ3RMa0hSSzBnVDg/view?usp=sharing

**Comment by abeloin** [comment]
@cyberundertaker Don't remove anything else

**Comment by abeloin** [comment]
Will try to add some code to better debug why the thread stop

**Comment by cyberundertaker** [comment]
@abeloin ok, i won't be home for few days(2-3) and SR will be off, so when i come back we can try again :) have a good weekend

**Comment by Kwart1001** [comment]
Just found this thread. I have the same problem as randmvilnce, DailyPlanet4 and abeloin. I will now try the suggestions and report back.

**Comment by fernandog** [comment]
@cyberundertaker any news?

**Comment by randmvilnce** [comment]
FYI, I have updated on sunday night to the latest version,  everything seems ok so far, all my shows downloaded last night anyways..
Ill post back if anything changes for me.
Thanks for the assistance.

**Comment by cyberundertaker** [comment]
@fernandog i'll be home in about 5 hours so will try again after update. Will post the results later.

**Comment by Morphy99** [comment]
Think I have a similar problem, shows from 27/03 haven't downloaded. I have debug enabled but I just get: "SEARCHQUEUE-BACKLOG-272644 :: Beginning backlog search for: [12 Monkeys]" and nothing after.
I'm on windows and python 2.7.8

**Comment by cyberundertaker** [comment]
@fernandog @abeloin doing the update right now... seems to be stuck for 10 minutes at orphan black:

2015-03-31 18:56:17 INFO     SHOWQUEUE-REFRESH :: 260315: Loading all episodes from the show directory E:\Serije E\Orphan Black
2015-03-31 18:56:17 INFO     SHOWQUEUE-REFRESH :: Performing refresh on Orphan Black
2015-03-31 18:56:14 INFO     SHOWQUEUE-FORCE-UPDATE :: 260315: Loading all episodes from theTVDB..
2015-03-31 18:56:11 INFO     SHOWQUEUE-FORCE-UPDATE :: Loading all episodes from the DB
2015-03-31 18:56:00 INFO     SHOWQUEUE-FORCE-UPDATE :: 260315: Loading show info from IMDb
2015-03-31 18:55:59 INFO     SHOWQUEUE-FORCE-UPDATE :: 260315: Loading show info from theTVDB
2015-03-31 18:55:59 INFO     SHOWQUEUE-FORCE-UPDATE :: Beginning update of Orphan Black

Debug log for Orphan Black:
http://pastebin.com/5kEBZXC7

**Comment by Kwart1001** [comment]
I still have the problem. There was an update from Sickrage master, but that didn't help me. The log is still empty if filtered on info->Show updater. I run Python27 on w7 x64

**Comment by fernandog** [comment]
@goatbased

**Comment by abeloin** [comment]
!!!DO NOT USE IN PRODUCTION!!! I AM NOT RESPONSIBLE FOR IF YOUR SICKRAGE GET CORRUPTED WITH THIS BRANCH. Make a backup

For those who want to try, here is an experimental branch testing with a status page that may help with the issue:
https://github.com/abeloin/SickRage/tree/feature-scheduler_fix

Status page in Config->Scheduler Status
![Imgur](http://i.imgur.com/esZitlY.png)

**Comment by fernandog** [comment]
@scythe000

**Comment by scythe000** [comment]
Aha...

**Comment by abeloin** [comment]
@scythe000 Should be in v4.0.16(next week) if everything goes according to plan, if not v4.0.17(in 2 weeks)

**Comment by goatbased** [comment]
Hey fellas, forgive me if I'm being dense but I'm having trouble tracking this thread and still trying to get the hang of Git.

Has a PR been issued to resolve the original issue here? Was it in yesterday's update? Anyone know of any workarounds to get me by until showuodater is working again?

Many thanks!

**Comment by abeloin** [comment]
@goatbased No PR issued, there is a feature branch being developed which will help for those kind of issue.

**Comment by abeloin** [comment]
Ok new PR(feature-scheduler_fix) pushed to develop:
https://github.com/SiCKRAGETV/SickRage/pull/1722

To test switch to develop

**Comment by scythe000** [comment]
I just switched to develop, and /config/status/ doesn't exist.

**Comment by abeloin** [comment]
It's now in home/status or via the "wrench and screwdriver"->server status

**Comment by scythe000** [comment]
Thanks! Here's a screenshot:
![new bitmap image](https://cloud.githubusercontent.com/assets/1904522/7014004/f8edb28a-dc73-11e4-8e06-2a7636f59e7b.jpg)

**Comment by scythe000** [comment]
But it's definitely not updating show scheduling and episode info.

**Comment by abeloin** [comment]
@scythe000 Set your show update to 16 and in 15 minutes it should start a force refresh and update

**Comment by abeloin** [comment]
And at 16h00-01 refresh the page you should see the items in the show queue part

**Comment by scythe000** [comment]
ok, done. I'll check back in then.

**Comment by abeloin** [comment]
The alive column is the one to check because if go to false(red) it mean the thread is dead and any jobs associated to that scheduler won't be done.

**Comment by scythe000** [comment]
Didn't do anything.

**Comment by abeloin** [comment]
Nothing in the show queue after a F5?

**Comment by scythe000** [comment]
nope

**Comment by scythe000** [comment]
![server](https://cloud.githubusercontent.com/assets/1904522/7014195/a2bf308e-dc76-11e4-81d3-ed94ce28f8b2.jpg)

**Comment by scythe000** [comment]
![vikings](https://cloud.githubusercontent.com/assets/1904522/7014203/b239e432-dc76-11e4-9f40-8e166b9e69ca.jpg)

**Comment by abeloin** [comment]
Can you send sickrage.log?

**Comment by scythe000** [comment]
https://www.dropbox.com/s/rybgfnkrz5tit7v/sickrage.log?dl=0

**Comment by abeloin** [comment]
No logs after 15:46?

**Comment by scythe000** [comment]
That's the log file

**Comment by abeloin** [comment]
In viewlog, switch to debug do you see something after 15:46?

**Comment by scythe000** [comment]
at debug, this is the most recent lines:
`2015-04-06 15:46:43 DEBUG    Thread-59 :: Empty lookup result - no XEM data for show 260449 on theTVDB
2015-04-06 15:46:43 DEBUG    Thread-59 :: Looking up XEM scene mapping using for show 260449 on theTVDB
AA
AAAttributeError: 'Config' object has no attribute 'status'
AA    method = getattr(self, route)
AA  File "C:\SickRage\sickbeard\webserve.py", line 241, in get
2015-04-06 15:40:19 DEBUG    TORNADO :: Failed doing webui request "status": Traceback (most recent call last):`

**Comment by abeloin** [comment]
Can you restart SR?

**Comment by scythe000** [comment]
Done.

http://pastebin.com/NdUULEpR

**Comment by abeloin** [comment]
Nothing in the status page?

**Comment by scythe000** [comment]
nope

**Comment by scythe000** [comment]
![status](https://cloud.githubusercontent.com/assets/1904522/7014454/e790a97e-dc79-11e4-8940-cd8b50411b64.jpg)

**Comment by abeloin** [comment]
* Stop SR
* open sickbeard/scheduler.py
* Edit line 38:
From:
`temp_now = datetime.datetime.now() + cycleTime`
to
`temp_now = datetime.datetime.now() - datetime.timedelta(hours=2) + cycleTime`
* Save the changes
* Restart SR and check the status page

**Comment by scythe000** [comment]
no change

**Comment by abeloin** [comment]
The last is 14h00 instead of 16h00?

**Comment by scythe000** [comment]
Last what?

**Comment by scythe000** [comment]
Start time still shows at 16:00

**Comment by abeloin** [comment]
ok replace gui/slick/interfaces/default/status.tmpl by this version https://gist.github.com/abeloin/2827d8be269a568b6de7

No need to restart, just refresh or open home/status page and it should force a show update.

Check in viewlog

**Comment by scythe000** [comment]
Nothing. Last update in log at debug level was this:
`2015-04-06 16:34:47 DEBUG    CHECKVERSION :: cur_commit = 78fe58e34961026738a90544e11aa4b7308d6670, newest_commit = 78fe58e34961026738a90544e11aa4b7308d6670, num_commits_behind = 0, num_commits_ahead = 0`

**Comment by abeloin** [comment]
And the last run hasn't changed to 2015-04-06 14:00:00?

**Comment by scythe000** [comment]
nope

**Comment by abeloin** [comment]
This is very strange, because when you refresh the page it send 2 commands to force a show update which your SR ignore

**Comment by abeloin** [comment]
Is it possible to access your system via teamviewer?

**Comment by scythe000** [comment]
Hey I'm about to go out to dinner with the family, when would be good for you? I am on Pacific time, which is GMT -8.

Sent from my iPhone

On Apr 6, 2015, at 5:15 PM, Alexandre Beloin <notifications@github.com<mailto:notifications@github.com>> wrote:


Is it possible to access your system via teamviewer?

**Comment by abeloin** [comment]
Eastern here(GMT-4) so it's currently 20:26. Around 22:00 my time?

**Comment by scythe000** [comment]
That should be fine. I'll try to be back by then. :-)

Sent from my iPhone

On Apr 6, 2015, at 5:28 PM, Alexandre Beloin <notifications@github.com<mailto:notifications@github.com>> wrote:


Eastern here(GMT-4) so it's currently 20:26. Around 22:00 my time?

**Comment by abeloin** [comment]
If not tommorow

**Comment by goatbased** [comment]
Hey guys, I just grabbed the develop branch to have a look at this, spun up a test VM, set the show update time to 18:00 and watched it kick off a full update without issue. I haven't inspected the results of the update yet but it looks promising so far. I suppose this isn't an apples to apples comparison since it's not on the machine that's been giving me grief, but it looks to be working on this test box.

It's finished processing all of the shows marked as "Ended" and is now on to shows that are continuing.

**Comment by goatbased** [comment]
Another update. One thing that I've noticed is that the update seems to hang every single time on a specific show, both in master, and now also in develop.  Relevant info is:

2015-04-06 18:14:31 INFO     SHOWQUEUE-FORCE-UPDATE :: Loading all episodes from the DB
2015-04-06 18:14:26 INFO     SHOWQUEUE-FORCE-UPDATE :: 71656: Loading show info from IMDb
2015-04-06 18:14:26 INFO     SHOWQUEUE-FORCE-UPDATE :: 71656: Loading show info from theTVDB
2015-04-06 18:14:26 INFO     SHOWQUEUE-FORCE-UPDATE :: Beginning update of Peep Show

I just watched this happen--and it now appears to be totally hung again (though the "Server Status" page would lead me to believe otherwise) in both my master (which I performed mass update on all shows earlier) and dev (which I let organically hit the update time at 18:00)

Things were looking good and then this:

![4-6-2015 6-23-21 pm](https://cloud.githubusercontent.com/assets/11284022/7015542/1957c054-dc8a-11e4-8583-799908cf6048.jpg)


And now this:
![4-6-2015 6-17-30 pm](https://cloud.githubusercontent.com/assets/11284022/7015497/8c9e11b8-dc89-11e4-9bf8-feb0c98a4eca.jpg)

**Comment by scythe000** [comment]
I'll be home in about 10 minutes. What's the best way for me to send you the login information?

Sent from my iPhone

On Apr 6, 2015, at 5:28 PM, Alexandre Beloin <notifications@github.com<mailto:notifications@github.com>> wrote:


Eastern here(GMT-4) so it's currently 20:26. Around 22:00 my time?

**Comment by abeloin** [comment]
Via sickrage.tv forum, user funkylove

**Comment by scythe000** [comment]
ok, PM sent

**Comment by abeloin** [comment]
Ok, the issue is with the function network_timezones.update_network_dict() in sickbeard/showUpdater.py#46. Commenting the line allow showUpdater to continue.

Look like requests get stuck

**Comment by scythe000** [comment]
Truly a masterful and epic debug session by Mr. Funkylove.He spent almost 2 hours performing delicate brain surgery on my install, and he now knows where to look for the root cause. Thank you, sir!

**Comment by scythe000** [comment]
@abeloin, the ShowQueue has been stuck on Downton Abbey since about 15 minutes or so after our remote session last night.

**Comment by scythe000** [comment]
Cancel that...restarted SR and it all processed.

**Comment by scythe000** [comment]
However, even though it went through the queue and added more broadcast dates, it still hasn't pulled the episode names and info for Vikings...

**Comment by abeloin** [comment]
@scythe000 Please post the logs, so we can check what was updated when Vikings was refreshed/updated

**Comment by scythe000** [comment]
https://www.dropbox.com/s/rybgfnkrz5tit7v/sickrage.log?dl=0

**Comment by Morphy99** [comment]
The last two weeks have been painful without sickrage snatching anything. Shows how much I rely on it! So I backed up my config and deleted program directory then did a fresh git pull now sickrage is working again. Please dont break it again!

**Comment by Kwart1001** [comment]
@Morphy, spoken as a true paying and unsatisfied customer, lol

**Comment by abeloin** [comment]
@Morphy99 
![Imgur](http://i.imgur.com/9RnZDoG.jpg)

**Comment by scythe000** [comment]
Hahahahahaha

Sent from my iPhone

On Apr 7, 2015, at 8:07 PM, Alexandre Beloin <notifications@github.com<mailto:notifications@github.com>> wrote:


@Morphy99<https://github.com/Morphy99>
[Imgur]<https://camo.githubusercontent.com/921b0acb89ac868602a3c97561ceba82ac165169/687474703a2f2f692e696d6775722e636f6d2f39526e5a446f472e6a7067>

**Comment by abeloin** [comment]
@scythe000 The provided log file had nothing regarding show refresh/update for Vikings, it must be in the other log files.

**Comment by abeloin** [comment]
@goatbased Sorry for the late reply, show update will add all the shows to the show queue and then exit so it's normal for it to be Active=False.

If the show queue is stuck, no other element will be processed form the queue, so if the number of item in the show queue isn't going down a few minutes later then something is stuck.

I'm planning on adding the current processed queue item name in the near future.

**Comment by scythe000** [comment]
![screen](https://cloud.githubusercontent.com/assets/1904522/7047283/66e3748a-ddc0-11e4-8a7b-8eb9d45563d2.jpg)
Looks like the show queue is stuck? if they were added 10 hours ago and haven't updated?

**Comment by abeloin** [comment]
@scythe000 Ok, I have to implement what is the current thread.

BTW there is some update in develop that was posted earlier, so please update

**Comment by scythe000** [comment]
Updating cleared that queue, but vikings still does not show the new episode info.

**Comment by scythe000** [comment]
So...the queue processes, but it's not actually grabbing all the new info from thetvdb...

**Comment by scythe000** [comment]
2015-04-08 18:59:33 SEARCHQUEUE-BACKLOG-260449 :: Check scene exceptions update failed. Unable to get URL: http://midgetspy.github.io/sb_tvdb_scene_exceptions/exceptions.txt
2015-04-08 18:59:33 SEARCHQUEUE-BACKLOG-260449 :: Check scene exceptions update failed. Unable to get URL: https://raw.githubusercontent.com/echel0n/sb_tvrage_scene_exceptions/master/exceptions.txt
2015-04-08 18:59:33 SEARCHQUEUE-BACKLOG-260449 :: [Rarbg] :: Unable to connect to Rarbg provider: error ('Connection aborted.', gaierror(11002, 'getaddrinfo failed'))
2015-04-08 18:59:30 SEARCHQUEUE-DAILY-SEARCH :: [Rarbg] :: Unable to connect to Rarbg provider: error ('Connection aborted.', gaierror(11002, 'getaddrinfo failed'))
2015-04-08 18:59:27 MAIN :: Check scene exceptions update failed. Unable to get URL: http://midgetspy.github.io/sb_tvdb_scene_exceptions/exceptions.txt
2015-04-08 18:59:27 MAIN :: Check scene exceptions update failed. Unable to get URL: https://raw.githubusercontent.com/echel0n/sb_tvrage_scene_exceptions/master/exceptions.txt
2015-04-08 18:59:27 MAIN :: Check scene exceptions update failed for theTVDB, Unable to get URL: http://thexem.de/map/allNames?origin=tvdb&seasonNumbers=1
2015-04-08 18:59:27 MAIN :: Check scene exceptions update failed for TVRage, Unable to get URL: http://thexem.de/map/allNames?origin=rage&seasonNumbers=1

**Comment by abeloin** [comment]
Updated develop: now show the currently running show queue

**Comment by abeloin** [comment]
@scythe000 Connection aborted.', gaierror(11002, 'getaddrinfo failed') mean unable to translate the dns name. DNS down?

**Comment by scythe000** [comment]
My internet seems fine

Sent from my iPhone

On Apr 8, 2015, at 7:19 PM, Alexandre Beloin <notifications@github.com<mailto:notifications@github.com>> wrote:


@scythe000<https://github.com/scythe000> Connection aborted.', gaierror(11002, 'getaddrinfo failed') mean unable to translate the dns name. DNS down?

**Comment by goatbased** [comment]
@abeloin FWIW, it's working flawlessly for me. I flipped my production environment to develop and things updated perfectly overnight at 3am just like I asked it to.

I did discover that the whole operation was hanging at the show "Peep Show" on my test VM due to something wacky happening in the cache. I shut SR down, cleared the cache folder by hand. After staring back up and initiating an update, all shows updated perfectly.

@scythe000 I'm not sure if you use a VPN service on this box, but I have noticed that, from time to time, PIA requires a renegotiation of the VPN for one reason or another. If you're sending your DNS requests across the tunnel, it could be having trouble there.

**Comment by scythe000** [comment]
No VPN for me :-)

Sent from my iPhone

On Apr 8, 2015, at 7:33 PM, goatbased <notifications@github.com<mailto:notifications@github.com>> wrote:


@abeloin<https://github.com/abeloin> FWIW, it's working flawlessly for me. I flipped my production environment to develop and things updated perfectly overnight at 3am just like I asked it to.

I did discover that the whole operation was hanging at the show "Peep Show" on my test VM due to something wacky happening in the cache. I shut SR down, cleared the cache folder by hand. After staring back up and initiating an update, all shows updated perfectly.

@scythe000<https://github.com/scythe000> I'm not sure if you use a VPN service on this box, but I have noticed that, from time to time, PIA requires a renegotiation of the VPN for one reason or another. If you're sending your DNS requests across the tunnel, it could be having trouble there.

**Comment by abeloin** [comment]
@scythe000 Can you stop SR, empty sickrage\cache\sessions and restart

**Comment by scythe000** [comment]
Done. No change. Also, it says there are 2 updates and I can't update because I have local changes (from your tweaks, i think). Should I git reset?

**Comment by abeloin** [comment]
Yes

**Comment by scythe000** [comment]
Okay, I'll do that as soon as I get back from band practice.

**Comment by Morphy99** [comment]
@kwart1001 was meant as a back handed compliment and merely emphasising what a great tool SR is and how much one can rely on it. No dissatisfaction here! Nice comeback anyway @abeloin lol :)

**Comment by abeloin** [comment]
@Morphy99 Glad to here you had a good laugh for my reply, it was the intended reaction I was trying to produce as I wasn't sure if your message was serious or not.

**Comment by abeloin** [comment]
@scythe000 Trakt enable?

**Comment by scythe000** [comment]
Ok, did reset and it updated. Show is still not up to date.

**Comment by Kwart1001** [comment]
@Morphy99 and @abeloin. 

"A person can make a fire with some wood and a backhanded compliment" :

I'm glad it's sorted out, keep up the good work ):)

I

**Comment by abeloin** [comment]
@scythe000 Can you try this:
* Enable show update at startup in general
* Stop SR
* Rename cache folder to cache-old
* Start SR
* Look at the status page

**Comment by scythe000** [comment]
Ok, showqueue is running..

**Comment by abeloin** [comment]
Can you send me a dir listing of the old cache/sessions folder?

**Comment by scythe000** [comment]
Directory of C:\SickRage\cache-old\sessions

[.]
[..]
79215acd3133a26ca88822f79c618c1baad2fb48d8c72b798552be5c
dir.txt
e91434754df11878cbe85c86474937fd5be26549b654feda9d90125f
[eeb49e23f8a00d496313794545bb459bc7f2339ce15e1528da23fc8c.lock]
               3 File(s)         11,297 bytes
               3 Dir(s)  74,144,567,296 bytes free

**Comment by scythe000** [comment]
That did it! I've got like 10 episodes downloading now, and all shows are updated.

**Comment by abeloin** [comment]
For some reason, the lockfile stay and cause this issue

**Comment by scythe000** [comment]
Yeah I saw that as soon as you had be do a directory output.

**Comment by scythe000** [comment]
That's one problem down :-)

**Comment by abeloin** [comment]
Yes

**Comment by abeloin** [comment]
@randmvilnce @cyberundertaker Please update to the latest master version

**Comment by cyberundertaker** [comment]
@abeloin mine is still stuck :(

**Comment by abeloin** [comment]
@cyberundertaker Deleted the cache/sessions folder?

**Comment by cyberundertaker** [comment]
@abeloin yes. deleted cache/sessions folder, updated SR to latest version and it gets stuck at walking dead:
![sr](https://cloud.githubusercontent.com/assets/11670746/7160691/f88a9452-e38a-11e4-9366-c5bffcb8f499.PNG)

LOG: https://drive.google.com/file/d/0ByFb16E-5XuQUFpGZUZyWWZjQUU/view?usp=sharing

**Comment by fernandog** [comment]
@cyberundertaker check logs for waking dead. See in which episode it's stucked

**Comment by cyberundertaker** [comment]
2015-04-16 15:37:38 INFO     SEARCHQUEUE-DAILY-SEARCH :: Seeing if we need anything from The Walking Dead
2015-04-16 15:15:53 DEBUG    SHOWQUEUE-REFRESH :: Name The.Walking.Dead.S05E08.REPACK.HDTV.x264-KILLERS gave release group of KILLERS, seems valid
2015-04-16 15:15:53 DEBUG    SHOWQUEUE-REFRESH :: Parsed The.Walking.Dead.S05E08.REPACK.HDTV.x264-KILLERS into The Walking Dead - S5E8 [ANIME VER: -1] [GROUP: KILLERS] [ABD: False] [ANIME: False] [whichReg: ['standard']]
2015-04-16 15:15:53 DEBUG    SHOWQUEUE-REFRESH :: Setter sets location to E:\Serije E\The Walking Dead\Season 05\The.Walking.Dead.S05E08.REPACK.HDTV.x264-KILLERS.mp4
2015-04-16 15:15:53 DEBUG    SHOWQUEUE-REFRESH :: 153021: E:\Serije E\The Walking Dead\Season 05\The.Walking.Dead.S05E08.REPACK.HDTV.x264-KILLERS.mp4 parsed to The Walking Dead 5x8
2015-04-16 15:15:53 DEBUG    SHOWQUEUE-REFRESH :: Parsed E:\Serije E\The Walking Dead\Season 05\The.Walking.Dead.S05E08.REPACK.HDTV.x264-KILLERS.mp4 into The Walking Dead - S5E8 [ANIME VER: -1] [GROUP: KILLERS] [ABD: False] [ANIME: False] [whichReg: ['standard', 'season_only']]
2015-04-16 15:15:52 DEBUG    SHOWQUEUE-REFRESH :: 153021: Creating episode object from E:\Serije E\The Walking Dead\Season 05\The.Walking.Dead.S05E08.REPACK.HDTV.x264-KILLERS.mp4
2015-04-16 15:15:52 DEBUG    SHOWQUEUE-REFRESH :: 153021: Creating episode from E:\Serije E\The Walking Dead\Season 05\The.Walking.Dead.S05E08.REPACK.HDTV.x264-KILLERS.mp4

**Comment by cyberundertaker** [comment]
@fernandog @abeloin 

ok, so i removed the walking dead from SR library and now everything is updating normally. it takes about 4 hours to complete all shows, so it might be a good idea to speed up the updater or to add the option to update only continuing shows, or set continuing shows as priority update. I would prefer the option to choose what shows to update or totally separate update of continuing and ended shows. (if necesary i can post a FR for this)

the mystery remains why SR was stuck at the walking dead.
but so far so good and i appreciate you help with the problem.

**Comment by abeloin** [comment]
Develop merge in master please update

**Comment by fernandog** [comment]
@randmvilnce please report back if issue was fixed

**Comment by randmvilnce** [comment]
My issues were resolved about 2 weeks back. Thank you.

**Comment by scythe000** [comment]
I'm back to having this issue. I opened #1778 .
